docs/_static/css/light.css

```diff
@@ -0,0 +1,31 @@
+/* links */
+.rst-content a{
+    color: #002fa7;
+}
+
+.rst-content a:visited {
+    color: #002fa7;
+}
```

Comment:
I think the visited color was meant to be identical to normal color only in the menu. Currently links in the text do get highlighted when visited and I'd personally count that as a feature :)

> One thing that is not optimal are the "visited" hyperlinks in the menu on the left. The menu items should not change their color to light blue when they were clicked/visited, they should keep the same color (that applies for both light mode and dark mode).

BTW, I can understand why designers might not like menu items having different colors when visited but personally I find that feature very useful so I'd keep the visited colors even in the menu. The light blue was definitely too flashy but maybe we could have at least a subtle color change? What do you think @franzihei?